integration/test.ats

```diff
@@ -78,36 +76,46 @@ def tolerance_test(name, input_file, num_mpi_tasks=1, tolerance=0.00001):
     # _serac_tolerance_test_end
     """
 
-
-    output_dir = os.path.abspath(name)
-
-    # Add driver test
-    test_args = "-i {0} -o {1}".format(input_file, output_dir)
-    my_test = test(
-         executable=serac_path, 
-         clas=test_args,
-         label=name,
-         np=num_mpi_tasks)
-
-    # Add checker for driver test
-    baseline_summary_file = os.path.abspath("{0}.json".format(name))
+    output_dir = os.path.abspath(name) + "_output"
     test_summary_file = "{0}/summary.json".format(output_dir)
+    baseline_summary_file = os.path.abspath("{0}.json".format(name))
 
-    checker_args  = "--baseline={0} --test={1}".format(baseline_summary_file, test_summary_file)
-    checker_args += " --tolerance={0}".format(tolerance)
-    my_checker = testif(
-               my_test,
-               executable=serac_tolerance_checker,
-               clas=checker_args,
-               label=name,
-               nosrun=True)
+    if not should_rebaseline:
+        # Add driver test
+        test_args = "-i {0} -o {1}".format(input_file, output_dir)
+        my_test = test(
+            executable=serac_path, 
+            clas=test_args,
+            label=name,
+            np=num_mpi_tasks)
+
+        # Add checker for driver test
+        checker_args  = "--baseline={0} --test={1}".format(
+            baseline_summary_file, test_summary_file)
+        checker_args += " --tolerance={0}".format(tolerance)
+        my_checker = testif(
+            my_test, 
+            executable=serac_tolerance_checker,
+            clas=checker_args,
+            label=name,
+            nosrun=True)
+    else:
+        # Rebaseline
+        if os.path.exists(test_summary_file):
+            print("Rebaselining {0}".format(name))
+            shutil.copyfile(test_summary_file, baseline_summary_file)
+        else:
+            print("Warning: tried to copy test file ({0}) that doesn't exist".format(
```

Comment:
avoid putting paths in the middle of a warning message. it makes it hard to read.  do something like this:

```
Warning: Something has occurred with file:
  <path>
```